From 8d9a88c2f33f8b6aaf8d655cbd68e3a3a0bb7ffa Mon Sep 17 00:00:00 2001
From: Michael Glembotzki <Michael.Glembotzki@iris-sensing.com>
Date: Thu, 7 Dec 2023 10:47:39 +0100
Subject: [PATCH] Still append bootloader string -iris_1.X.X to u-boot

The uboot version suffix  UBOOT_LOCALVERSION is no longer part of common
uboot builds. To fix the latest changes to iris needs, we patch the
version string.

Signed-off-by: Michael Glembotzki <Michael.Glembotzki@iris-sensing.com>
---
 classes/fsl-u-boot-localversion.bbclass | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/classes/fsl-u-boot-localversion.bbclass b/classes/fsl-u-boot-localversion.bbclass
index 94b30c04..9e1a29bf 100644
--- a/classes/fsl-u-boot-localversion.bbclass
+++ b/classes/fsl-u-boot-localversion.bbclass
@@ -29,8 +29,8 @@ do_compile:prepend() {
                         head=`git --git-dir=${S}/.git rev-parse --verify --short $hash 2> /dev/null`
                 fi
                 patches=`git --git-dir=${S}/.git rev-list --count $head..HEAD 2> /dev/null`
-                printf "%s%s%s%s" +g $head +p $patches > ${S}/.scmversion
-                printf "%s%s%s%s" +g $head +p $patches > ${B}/.scmversion
+                printf "%s%s%s%s%s" "${UBOOT_LOCALVERSION}" +g $head +p $patches > ${S}/.scmversion
+                printf "%s%s%s%s%s" "${UBOOT_LOCALVERSION}" +g $head +p $patches > ${B}/.scmversion
         else
 		printf "%s" "${UBOOT_LOCALVERSION}" > ${S}/.scmversion
 		printf "%s" "${UBOOT_LOCALVERSION}" > ${B}/.scmversion
-- 
2.42.0

